Adding a blog post "Protocol for membership" (#3938)
unlock-protocol/unlock: unlock-protocol.com/blog/protocol-for-membership.md

---
title: A Protocol for Membership
subTitle: Monetizing creation by leveraging membership to communities
authorName: Julien Genestoux
publishDate: June 19, 2019

> description: The unlock protocol provides a way for people to show their support and access exclusive content, features, or even status. It is a web-native way for creators to monetize their work.

Reviewer: ```suggestion
description: The Unlock protocol provides a way for people to show their support and access exclusive content, features, or even status. It is a web-native way for creators to monetize their work.
```

image: /static/images/blog/protocol-for-memberships.png
---

> The difference between the web and native applications is the presence of **links**. These links are the reason why the web is a "network" as they connect pages together, making a graph. Google and its pagerank algorithm have leveraged that graph to rank wesbites, by assuming that the sites which have the most links are the most relevant.

Reviewer: ```suggestion
The difference between the web and native applications is the presence of **links**. These links are the reason why the web is a "network", as they connect pages together, making a graph. Google and its PageRank algorithm have leveraged that graph to rank websites, by assuming that the sites which have the most links are the most relevant.
```

![Use one Unlock lock for a variety of services and products.](/static/images/blog/hero.jpg)

## Memberships > Attention

> However, since the begining, these links have also formed **communities**, as groups of sites linking to one another. Websites which discuss the same topics tend to link to each other, creating de-facto communities. The social web re-inforced that trend: people following each other formed more communities. Creators gathered groups of users around their work. Whether they are blogger or journalists, musician or even developers, they formed groups which attracted users because of their values, the content they produce, or even the tools that were offered to the members of the community.

Reviewer: ```suggestion
However, since the beginning, these links have also formed **communities**. Websites which discuss the same topics tend to link to each other, creating de-facto communities of interest. The social web reinforced that trend: people following each other formed more communities. Creators gathered groups of users around their work. Whether they are blogger or journalists, musician or even developers, they formed groups which attracted users because of their values, the content they produce, or even the tools that were offered to the members of the community.
```

> Up until now, _we monetized the web through attention_. The creators who were able to "steal" the most would be paid the most. We're obviously reaching the limits of that model. It is time to start considering that these communities have value for their members... and that these members actually want to contribute to the community as well.

Reviewer: ```suggestion
Up until now, _we monetized the web through attention_. The creators who were able to "steal" the most would be paid the most. We're obviously reaching the limits of that model. It is time to start considering that these communities have value for their members... and that these members actually want to contribute to their communities as well.
```

## A protocol for memberships

Unlock is a protocol for memberships. Creators have locks which define the terms of the membership (price, duration, quantity, or even currency!) and consumers can purchase keys to these locks.

> In practice, that means that the protocol empowers creators (or anyone, really!) to create locks for their communities. It's a protocol so that anyone can participate: there is no gatekeeper or middlemen.

Reviewer: ```suggestion
In practice, that means that the protocol empowers creators (or anyone, really!) to create locks for their communities. It's an open protocol, so anyone can participate: there is no gatekeeper or middleman.
```

Locks themselves can be used for many different things, limiting access to content (paywall...), to features (software licenses...), or even status (patronage...).

## Leading by example!

> We're putting [our money where our mouth is](/blog/crypto-first/). Starting today, we added a lock to this website. We decided to **limit access to the comments** to the members of the community.

Reviewer: ```suggestion
We're putting [our money where our mouth is](/blog/crypto-first/). Starting today, we've added a lock to this website. We decided to **limit access to the comments** to the members of the community.
```
